pyproject.toml

```diff
@@ -21,6 +21,7 @@ dependencies = [
 description = "Clinical study SQL generation for data derived from bulk FHIR"
 readme = "README.md"
 license = { text="Apache License 2.0" }
+homepage = "https://smarthealthit.org/cumulus-a-universal-sidecar-for-a-smart-learning-healthcare-system/"
```

Comment:
I think this needs to be in its own url section?

Reply:
this one has some legacy support but i can move it into the URL dict.

Reply:
Oh really that's nice

Reply:
i've actually got in the right place as well so i'm just removing this one.